SofaKernel/framework/sofa/helper/Utils.cpp

```diff
@@ -249,7 +249,7 @@ static std::string computeSofaPathPrefix()
         const std::string exePath = Utils::getExecutablePath();
         std::size_t pos = exePath.rfind("/bin/");
         if (pos == std::string::npos) {
-            msg_error("Utils::getSofaPathPrefix()") << "failed to deduce the root path of Sofa from the application path: (" << exePath << ")";
+            // msg_error("Utils::getSofaPathPrefix()") << "failed to deduce the root path of Sofa from the application path: (" << exePath << ")";
```

Comment:
Any reason to why removing this msg error? Since the line after it returns a wrong sofa prefix path because it is the "Safest thing to return in this case, I guess", we should notify the user of this wrongly path.

Or return an empty string and let the caller handle the error.

Reply:
Oh I just saw your previous comment, sorry.

I also got the seg fault while loading sofa in a python environment (without gui) on mac osx. I fear that commenting this line will simply "push" temporary this bug instead of solving it.

Reply:
By the way, this bug comes from msg_error called from all those cute sofa static calls when the application (or library) load in memory.

Reply:
Did you take some time to investigate the segfault?

Reply:
Oh ok. Any idea hox to fix it?

Reply:
It's on my to do list :)

Reply:
https://github.com/sofa-framework/sofa/issues/636 created